src/js/components/LeftNav.jsx

```diff
@@ -1,13 +1,20 @@
-import {applicationName} from '../../../config'
+import ActionHelp from 'material-ui/svg-icons/action/help'
 import Avatar from 'material-ui/Avatar'
 import {browserHistory} from 'react-router'
 import Divider from 'material-ui/Divider'
 import Drawer from 'material-ui/Drawer'
 import ListItem from 'material-ui/List/ListItem'
 import {logoStyle} from '../styles/common'
 import {routes} from '../constants'
+import {applicationName, helpMenuItems} from '../../../config'
 import React, {Component} from 'react'
 
+const openInNewTab = (url) => {
+  const win = window.open(url, '_blank')
+  
+  if (win) win.focus()
```

Comment:
Not particularly. generally, i reverse if statements so that there are explicit paths for each possibility, without unnecessarily using an `else` statement.

```javascript
if (!win) {
  return
}

win.focus()
```